lib/Service/src/SettingsHandler.cpp

```diff
@@ -65,7 +65,7 @@
 bool SettingsHandler::loadConfigurationFile(const String& filename)
 {
     bool                              isSuccessful  = false;
-    const uint32_t                    maxBufferSize = 1024U;
+    const uint32_t                    maxBufferSize = 2048U;
     StaticJsonDocument<maxBufferSize> doc;

```

Comment:
Hmm with 2kbyte it may make sense to put in on the heap with DynamicJsonDocument.